sync/class.jetpack-sync-module-themes.php

```diff
@@ -56,6 +64,96 @@ function expand_theme_data() {
 		return array( $this->get_theme_support_info() );
 	}
 
+	function sync_sidebar_widgets_actions( $old_value, $new_value ) {
```

Comment:
This isn't a blocking comment, but the sync_sidebar_widgets_actions() method is handling a number of widget action types. Consider handling each in a small method of its own that is called from sync_sidebar_widgets_actions().